traverson4j-jackson2/src/test/java/uk/co/autotrader/traverson/conversion/JacksonResourceConverterTest.java

```diff
@@ -4,15 +4,16 @@
 import com.google.common.io.Resources;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang3.reflect.FieldUtils;
+import org.hamcrest.BaseMatcher;
```

Comment:
Could we aim to drop hamcrest usage in favour of assertj assertions please? We should probably update the checkstyle to guide this going forward

Reply:
This is because Mockito reworked how ArguementMatchers worked and I couldnt work out how to do this any other way

Reply:
also .expect() method requires a Matcher.class (which is hamcrest)

Reply:
theyre gone now